expression/reconstructor/reconstructor.go

```diff
@@ -92,11 +100,15 @@ func unknownByteArrayPretty(bytes []byte) string {
 	return fmt.Sprintf("0x%s (str:%s)", hex.EncodeToString(bytes), strconv.Quote(string(bytes)))
 }
 
-func addressPretty(value []byte) string {
+func addressPretty(value []byte, bech32Addr ...bool) string {
```

Comment:
You can make addressPretty a method of ExprReconstructor to get access to additional configs, let's keep the arguments clean, to keep the refactoring in the other repos manageable.